### PR TITLE
Add CAXCtrl device

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

File: siriuspy/siriuspy/devices/beamline.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+"""Beamline devices."""
+
+from .device import Device as _Device
+
+
+class BeamLineMirror(_Device):
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/beamline.py:6]: I think this is a device very specific for CAX, the names of the PVs seems to be different for each beamline

File: siriuspy/siriuspy/devices/beamline.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    @property
+    def m1_pos_tx(self):
+        """Return mirror M1 linear position axis for Tx [mm]."""
+        return [':B.RBV']
+
+    @m1_pos_tx.setter
+    def m1_pos_tx(self, value):
+        """Set mirror M1 linear position axis for Tx [mm]."""
+        self[':B.RBV'] = value
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/beamline.py:47]: bug:  `RBV` is equivalent to -RB in accelerators PV naming standard

File: siriuspy/siriuspy/devices/beamline.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    @property
+    def m1_pos_y1(self):
+        """Return mirror M1 linear position y1 for Rx, Rz and Ty [mm]."""
+        return [':C.RBV']
+
+    @m1_pos_y1.setter
+    def m1_pos_y1(self, value):
+        """Set mirror M1 linear position y1 for Rx, Rz and Ty [mm]."""
+        self[':C.RBV'] = value
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/beamline.py:57]: idem

File: siriuspy/siriuspy/devices/beamline.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    @property
+    def m1_pos_y2(self):
+        """Return mirror M1 linear position y2 for Rx, Rz and Ty [mm]."""
+        return [':D.RBV']
+
+    @m1_pos_y2.setter
+    def m1_pos_y2(self, value):
+        """Set mirror M1 linear position y2 for Rx, Rz and Ty [mm]."""
+        self[':D.RBV'] = value
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/beamline.py:67]: idem

File: siriuspy/siriuspy/devices/beamline.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+    @property
+    def m1_pos_y3(self):
+        """Return mirror M1 linear position y3 for Rx, Rz and Ty [mm]."""
+        return [':E.RBV']
+
+    @m1_pos_y3.setter
+    def m1_pos_y3(self, value):
+        """Set mirror M1 linear position y3 for Rx, Rz and Ty [mm]."""
+        self[':E.RBV'] = value
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/beamline.py:77]: idem

File: siriuspy/siriuspy/devices/beamline.py (hunk 1 above)
Comment (xresende) [reply siriuspy/siriuspy/devices/beamline.py:6]: I thought they all followed a standard for mirror controls...

Comment (xresende) [reply siriuspy/siriuspy/devices/beamline.py]: See what you think now.

File: siriuspy/siriuspy/devices/beamline.py (hunk 2 above)
Comment (xresende) [reply siriuspy/siriuspy/devices/beamline.py]: yes. copy&paste mistake. thanks.

File: siriuspy/siriuspy/devices/beamline.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        super().__init__(devname, props2init=props2init, **kwargs)
+
+    @property
+    def mirror_m1_pos_ry(self):
+        """Return mirror M1 linear position axis for Ry [mm]."""
+        return [':A:PP01::A.RBV']
+
+    @mirror_m1_pos_ry.setter
+    def mirror_m1_pos_ry(self, value):
+        """Set mirror M1 linear position axis for Ry [mm]."""
```
Comment (fernandohds564) [inline siriuspy/siriuspy/devices/beamline.py:29]: I don't understand the docstrings. What do they mean?

Comment (xresende) [reply siriuspy/siriuspy/devices/beamline.py]: I updated docstrings. take a look

**Review by fernandohds564 — APPROVED** [review]

File: siriuspy/siriuspy/devices/beamline.py [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+        RotY is performed by a linear actuator in one of
+        longitudinal ends of the mirror. The mirror is pivoted
+        at its longitudinal center and the linear actuator induces
+        a rotation around the Y axis.
+        """
+        return [':A:PP01:A.RBV']
+
+    @mirror_m1_roty_pos.setter
+    def mirror_m1_roty(self, value):
```
Comment (vellosok75) [inline siriuspy/siriuspy/devices/beamline.py:45]: `mirror_m1_roty` -> `mirror_m1_roty_pos`

**Review by vellosok75 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by Gabrielrezende-asc — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Pull request merged**